embed.go

```diff
@@ -0,0 +1,61 @@
+package kcp
```

Comment:
I don't think we want this at the top level. Maybe just keep it in pkg/reconciler/cluster?

Reply:
Well, that the easiest way to be able to embed the content of the `/config` directory, since you cannot embed files at a lower level than the current package (`../` is not supported). And I'm not sure moving `/config` at a package level is a good idea either.

Reply:
Hmm, this is really unfortunate. 😞 

Maybe we could symlink `config/` to `pkg/reconciler/cluster/config/`? Does //go:embed` chase symlinks? 🤞

Reply:
I'm not a fan of the symlink solution TBF.
I forced pushed (in commit 80e1b86d5fef7c95a23de2bf7c29f0ec6038c2df) a fix that seems quite a compromise: just have an `embed.go` file at the root of the module, that only defines global variables, and use those variables in code placed in the regular packages.

Reply:
This is a good compromise, thanks for thinking of it. 👍